### DISCUSSION TITLE
Easy way to turn off device

### DISCUSSION THREAD

**Discussion by AndreZoro** [eduardofilo/RG350_adam_image#148] [Ideas & Requests]
Hi,
I know how to turn the device off, but I think it would be nice to use the hardware buttons as well.
Would be cool, if the main power button could turn off the device (maybe, using long press) and the reset button could put it to sleep, using a long press for the reset.
Just a suggestion, I am already happy that the volume rocker on my device got support in some version.

**Comment by eduardofilo** [comment]
As with SimpleMenu, I am not the developer of the system (OpenDingux beta in this case), so these requests should go to them. In any case, in the latest versions of ODb that are installed in Adam, precisely by holding down Power for a couple of seconds, a controlled shutdown is invoked. Try to hold a while longer than you do.
